### DISCUSSION TITLE
OsmAnd vs Google My Maps.

### DISCUSSION THREAD

**Discussion by pathexplorer** [osmandapp/OsmAnd#11795] [Ideas]
It would be great to turn OsmAnd into an alternative to Google My Maps.

Now I can create my points in Bookmarks, group them by categories, but at the same time, I cannot attach a photo to the bookmark. As a result, the database is purely textual. And what's the point then from the point called "Nice view" if I can't quickly see it? And how do you decide where to go to Interns Tree # 3, or Hilly Slope # 5?

To do this, you need to allow you to add your arbitrary photo to the points of the Bookmarks. This is partially implemented with Mappilary. But through this service it is inconvenient to add photos to a personal database of points, it requires a personal visit to the place, and will upload this photo to a public network, and in general, it is not clear why use an intermediary application
If you just create a photo note next to a point, then after a while it will overload the map with unnecessary points, prevent you from seeing audio notes nearby.

**Comment by sbmooc** [comment]
I'm looking to do the exact same. I know it's been a while, but did you manage to progress this at all, or find an alternative?

**Reply by pathexplorer** [reply]
At now, the situation is same
